packages/parser/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "mermaid-parser",
+  "version": "0.2.0",
```

Comment:
When should we rename it and deprecate the old one?

Reply:
> We might want to make the version 1.0.0 so that we can make MAJOR, MINOR, and PATCH releases.

Keeping it `0.` will allow us to make breaking changes easily, as it might be required when adding new diagrams. 
I think we should delay the v1 release until we're sure it's stable.

Reply:
> When should we rename it and deprecate the old one?

I'll publish it to the org before merging. Then you can deprecate the old one.

Reply:
> I'll publish it to the org before merging. Then you can deprecate the old one.

Sounds good! Remember to also update the `mermaid-parser` links in the README.md!